rules/no-ignored-test-files.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+var path = require('path');
+var pkgUp = require('pkg-up');
+var multimatch = require('multimatch');
+var util = require('../util');
+var createAvaRule = require('../create-ava-rule');
+
+var defaultFiles = [
+	'test.js',
+	'test-*.js',
+	'test/**/*.js'
+];
+
+function isInIgnoredFolder(relativeFilePath) {
+	var folders = relativeFilePath.split(path.sep);
+	var ignoredFolder;
+	if (folders.indexOf('fixtures') !== -1) {
+		ignoredFolder = 'fixtures';
+	} else if (folders.indexOf('helpers') !== -1) {
+		ignoredFolder = 'helpers';
+	}
+
+	if (ignoredFolder) {
+		return 'Test file is ignored because it is inside a `' + ignoredFolder + '` folder';
+	}
+
+	return null;
+}
+
+function isNotInGlob(files, relativeFilePath) {
+	if (multimatch([relativeFilePath], files).length === 0) {
+		return 'Test file is ignored because it is not in `' + files.join(' ') + '`';
+	}
+
+	return null;
+}
+
+function isIgnored(rootDir, files, filepath) {
+	var relativeFilePath = path.relative(rootDir, filepath);
+	return isInIgnoredFolder(relativeFilePath) ||
+		isNotInGlob(files, relativeFilePath);
+}
+
+function getPackageInfo() {
+	var packageFilePath = pkgUp.sync();
```

Comment:
Might be good to document the assumption that `process.cwd()` is the directory of the file being linted?

Some test suites may have nested `package.json` files. I wonder if you should keep searching until you find AVA config?

[Here's an example from AVA itself with a nested `package.json`](https://github.com/sindresorhus/ava/tree/005b656b9950ab3fa1e2bd9af326d5b39540e71b/test/fixture/caching).